src/data/accessibility.json

```
@@ -0,0 +1,55 @@
+            "The verification of compliance with the requirements of the Corona-Warn-App (iOS) is based on a self-evaluation carried out on 18.01.2022.",
+            "<b>The Corona-Warn-App (iOS) content listed below is not accessible for the following reasons:</b><ul><li>The content of the app is limited to portrait display, in contradiction to requirement 1.3.4 WCAG 2.1 or 11.1.3.4 EN 301 549 V3.2.1.</li><li>The presented contrast ratio for the visual presentation of text and images of text differs from the requirement 1.4.3 WCAG 2.1 or 11.1.4.3 EN 301 549 V3.2.1 in the case of the navigation bar caption texts.</li><li>Enlargement of text (without assistive technology by up to 200%) may result in lack of enlargement or truncation of text fields, in contradiction to requirement 1.4.4 WCAG 2.1 or 11.1.4.4.1 EN 301 549 V3.2.1.</li><li>The following functionality applies to physical keyboards connected to the end device. In contradiction to requirement 2.1.1 WCAG 2.1 and 11.2.1.1.1 EN 301 549 V3.2.1, keyboard usage is not possible on the \"Terms of use\" and \"Privacy\" pages. On the \"Edit people\" and \"Edit places\" pages, content cannot be deleted using the keyboard.</li><li>On the \"Edit People\" and \"Edit Places\" pages, the delete key cannot be exposed with a single pointer action, in contradiction to requirement 2.5.1 WCAG 2.1 or 11.2.5.1 EN 301 549 V3.2.1.</li></ul>",
+            "We are working to remove the listed barriers as soon as possible.",
+            "<b>Date of creation or last update of the declaration</b>",
+            "This statement was created on 21.02.2022."
+        ]
+    },
+    "section-report-barriers": {
+        "headline": {
+        "title": "Report barriers: Feedback contacts",
+        "text": "You would like to inform us about existing barriers or request information on the implementation of accessibility? You are welcome to contact us by <a href='coronawarnapp...rki...de' class='email'>e-mail</a> or via our <a href='https://www.rki.de/SharedDocs/Kontaktformulare/weitere/Corona-Warn-App/Corona-Warn-App_Integrator.html' target='_blank' rel='noopener noreferrer'>contact form</a>."
+        }
+    },
+    "section-arbitrary-service": {
+        "title": "Arbitration",
+        "textblock": [
+            "In case no satisfactory solution has been found even after your feedback to the above-mentioned contact, you can approach the arbitration service in accordance with § 16 BGG. The BGG arbitration service is responsible for supporting out-of-court dispute resolution in the event of conflicts on the topic of accessibility between people with disabilities and federal public entities. The arbitration service is free of charge. No legal counsel needs to be involved. For more information on the arbitration service and how to submit an application, please visit the <a href='https://www.schlichtungsstelle-bgg.de' target='_blank' rel='noopener noreferrer'>website of the arbitration service</a>.",
```

Link to English page.

```suggestion
            "In case no satisfactory solution has been found even after your feedback to the above-mentioned contact, you can approach the arbitration service in accordance with § 16 BGG. The BGG arbitration service is responsible for supporting out-of-court dispute resolution in the event of conflicts on the topic of accessibility between people with disabilities and federal public entities. The arbitration service is free of charge. No legal counsel needs to be involved. For more information on the arbitration service and how to submit an application, please visit the <a href='https://www.schlichtungsstelle-bgg.de/Webs/SchliBGG/EN/home' target='_blank' rel='noopener noreferrer'>website of the arbitration service</a>.",
```

---

@Ein-Tim Thanks.